grammar.js

```diff
@@ -240,6 +271,24 @@ module.exports = grammar({
 
         effective_date: $ => seq('=', $._single_date),
 
+        // https://ledger-cli.org/doc/ledger3.html#Period-Expressions
+        date_spec: $ => choice(
+            $._4d,
+            $._single_date,
+            $._month,
+            $._relative_date_spec,
+        ),
+
+        // HACK: months and abbreviated months are locale specific, so just
```

Comment:
That's an... interesting choice from ledger, given that all other words (e.g. those in `_interval_period`) must be English.

Reply:
Oh, you caught me! This was only an assumption during implementation and I didn't go back to validate it. It's not true, the ledger code for supported day/month names is here: https://github.com/ledger/ledger/blob/c215f6ecee54f340c68b59a71f5e59525634f504/src/times.cc#L187-L236